vcr/test/openid4vci_integration_test.go

```diff
@@ -190,6 +190,20 @@ func TestOpenID4VCIDisabled(t *testing.T) {
 		data, _ := io.ReadAll(resp.Body)
 		assert.Equal(t, `{"detail":"openid4vci is disabled","status":404,"title":"Operation failed"}`, string(data))
 	})
+	t.Run("Issues over network", func(t *testing.T) {
+		credential := testCredential()
+		credential.Issuer = walletDID.URI()
+		credential.ID, _ = ssi.ParseURI(walletDID.URI().String() + "#1")
+		credential.CredentialSubject = append(credential.CredentialSubject, map[string]interface{}{
+			"id":           walletDID.URI().String(),
+			"purposeOfUse": "test",
+		})
+
+		vcrService := system.FindEngineByName("vcr").(vcr.VCR)
+		_, err := vcrService.Issuer().Issue(audit.TestContext(), credential, true, false)
+
+		assert.ErrorContains(t, err, "unable to publish the issued credential")
```

Comment:
not really happy with this assertion, but I could not get NutsComm registered on localhost.

Reply:
We could always make it an e2e test